core/src/main/scala/com/salesforce/op/OpWorkflow.scala

```diff
@@ -521,12 +522,15 @@ class OpWorkflow(val uid: String = UID[OpWorkflow]) extends OpWorkflowCore {
     maxCorrelation: Double = 0.95,
     correlationType: CorrelationType = CorrelationType.Pearson,
     protectedFeatures: Array[OPFeature] = Array.empty,
-    textBinsFormula: (Summary, Int) => Int = RawFeatureFilter.textBinsFormula
+    protectedJSFeatures: Array[OPFeature] = Array.empty,
```

Comment:
1. how is `protectedJSFeatures` different from `protectedFeatures`?
2. please add docs for `protectedJSFeatures` param

Reply:
This was already enabled in `RawFeatureFilter`, but I didn't see it `OpWorkflow.withRawFeatureFilter`, just making sure it's here as well